torch/csrc/distributed/c10d/TCPStoreLibUvBackend.cpp

```diff
@@ -186,7 +186,7 @@ class UvTcpServer : public UvTcpSocket {
       int uv_res = uv_tcp_open((uv_tcp_t*)res->unsafeGetStream(), socket);
       TORCH_CHECK(
           uv_res == 0,
-          "Failed to open existing socket. socket:%d code:{} name:{} message:{}",
+          "Failed to open existing socket. socket:{} code:{} name:{} message:{}",
```

Comment:
If this uses libfmt, there should be a way to do proper formatting of the int within the `{}`. Minor nit though.

Reply:
I would like to use C++20 right now...